package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start:dev": "nodemon",
-    "start": "npm run build && node build/index.js",
+    "start": "npm run build && node dist/index.js",
```

Comment:
Should solve the prior issue regarding yarn start expecting creation of build folder 👍🏾👍🏾